test/e2e_node/image_list.go

```diff
@@ -61,11 +61,11 @@ var NodeImageWhiteList = sets.NewString(
 	"gcr.io/kubernetes-e2e-test-images/node-perf/tf-wide-deep-amd64:1.0",
 )
 
-// updateImageWhiteList updates the framework.ImageWhiteList with
+// updateImageAllowList updates the framework.ImageWhiteList with
 // 1. the hard coded lists
 // 2. the ones passed in from framework.TestContext.ExtraEnvs
 // So this function needs to be called after the extra envs are applied.
-func updateImageWhiteList() {
+func updateImageAllowList() {
 	// Union NodeImageWhiteList and CommonImageWhiteList into the framework image white list.
 	framework.ImageWhiteList = NodeImageWhiteList.Union(commontest.CommonImageWhiteList)
```

Comment:
many occurrences of "white" in comment above and variable name.

Reply:
Thanks for catching that!
I renamed ImageWhiteList from the framework but im leaving NodeImageWhiteList and CommonImageWhiteList for separate PRs, ref https://github.com/kubernetes/kubernetes/issues/92319 and https://github.com/kubernetes/kubernetes/issues/92320